README.md

```diff
@@ -40,10 +40,12 @@ See the [DBAD license](https://github.com/philsturgeon/dbad) in your language or
 
 | Name | Type | Description |
 | --- | --- | --- |
-| path | <code>String</code> | Path to folder awesome.js is located in |
-| bower | <code>String</code> | path to bower components |
+| awesome.path | <code>String</code> | Path to folder awesome.js is located in |
+| awesome.bower | <code>String</code> | path to bower components |
 | constants | <code>Object</code> | awesome constants |
 | config | <code>Object</code> | awesome config objects |
+<<<<<<< HEAD
+=======
 | configMerge | <code>function</code> | deep recursive merge for awesome config objects |
```

Comment:
these are conflicts and should never be comitted.

in this case all you have to do is rebuild the docs